apps/glusterfs/brick_allocate.go

```diff
@@ -28,6 +28,13 @@ func NewBrickSet(s int) *BrickSet {
 	return &BrickSet{SetSize: s, Bricks: []*BrickEntry{}}
 }
 
+func NewSparseBrickSet(s int) *BrickSet {
```

Comment:
Heh. I always get in trouble with names. It's not that the brick set is sparse at the gluster level, its that the brick set object in heketi starts off "sparse" in that the array holds nulls but is sized up to the expected brick set set, rather than being an empty array.

It's not arbiter specific because I had wanted something like this earlier for some other re-factoring I did, but I convinced myself not to do it then. I will try to see if I can come up with a better name quickly.